src/core/content/util.ts

```diff
@@ -496,7 +496,7 @@ export function extractImageUrlFromSelectors(
 		return null;
 	}
 	const elements = queryElements(selectors);
-	if (!elements) {
+	if (!elements || !elements.length) {
```

Comment:
Still learning this TS syntax - just to something more akin to `elements.length === 0

Reply:
Did it out of habit. It's essentially equivalent to elements.length === 0.
Its a bit of a more javascripty way of doing things, as if _somehow_ (javascript) we end up not actually having an array here it will still short circuit and not error.
I don't think it should be possible for there to not be an array at this point though